Assets/HotmaxEditorVR/Hittables/propTile.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class propTile : MonoBehaviour, IHittable
+{
+    //--local refs
+    private stateManager _stateManagerMutatorRef;
+
+    public int currentPage;
+
+    public GameObject spawnableProp;
+    public TextMesh propTileText;
+    private GameObject boundingScaleBox;
+
+    void Start()
+    {
+        _stateManagerMutatorRef = init._stateManagerMutatorRef;
+        displayPropInMenu(spawnableProp, currentPage);
+    }
+
+    public void receiveHit(RaycastHit hit)
+    {
+        //get the correct object from the current chunk
+        GameObject newProp = Instantiate(spawnableProp);
+        newProp.name = spawnableProp.name;
+        newProp.transform.parent = init.props.transform;
+        newProp.SetActive(true);
+
+        //set the spawned object to the new selected object
+        _stateManagerMutatorRef.SET_SELECTED_OBJECT(newProp);
+        newProp.transform.position = hit.point;
+        //check for prop -some objects are loaded with props, some aren't. 
```

Comment:
what is this check doing?

Reply:
or why would some thing have prop already and some not?

Reply:
maybe that was a weird artifact from primitives. just tested it again and seems fine without the check. will remove